### PR TITLE
fix: assets outputted multiple times

### PR DESCRIPTION
**Pull request by adrianthedev**
# Description

Fixes an issue where, sometimes, the asset files would be added multiple times.
Also fixes https://github.com/avo-hq/avo/issues/2978

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [documentation](https://github.com/avo-hq/avodocs)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/generators/avo/eject_generator.rb [hunk 1]
```diff
@@ -146,7 +146,7 @@ def eject_component(component_to_eject = options[:component], confirmation: true
             [dest_rb, dest_erb].each do |path|
               if component.starts_with?("avo/views/")
                 modified_content = File.read(path).gsub("Avo::Views::", "Avo::Views::#{options[:scope].camelize}::")
-              elsif component.starts_with?("avo/fields/")
+              elsif component.starts_with?("avo/fields/") && options["field-components"].present?
```
Comment (Paul-Bob) [inline lib/generators/avo/eject_generator.rb:149]: Is this fixing this issue? https://github.com/avo-hq/avo/issues/2978

**Review by Paul-Bob — APPROVED** [review]
Any idea why there are duplicated values inside the `@stylesheets` and `@javascripts` arrays? Can't we avoid the duplication from where the values are added to the array?

**Comment by adrianthedev** [comment]
That would be the best fix. This is a hotfix.

File: lib/generators/avo/eject_generator.rb (hunk 1 above)
Comment (Paul-Bob) [reply lib/generators/avo/eject_generator.rb:149]: I'm supposing it is

**Pull request merged**